docs/source/manual/health.rst

```diff
@@ -70,3 +70,21 @@ Usage
       return "foobar";
     }
   }
+
+* If your health dependency is a HTTP service, you can use the HttpHealthDependency_ class to declare your dependency.
```

Comment:
this is minor, but could you mention here that this comes from okhttp?

Reply:
I am not sure I understand this. Are you wanting me to mention that HttpHealthDependency class comes from OkHttp?

Reply:
No i wanted to mention in passing in the doc that we're using okhttp to perform the health check because i dont think we're using okhttp for very much else in beadledom

Reply:
Is that necessary though? Because thats an implementation detail

Reply:
I agree that how we are making the HTTP calls is an implementation detail and doesn't seem necessary to explain in the docs.